include/dynamixel_control_hw/dynamixel_loop.hpp

```diff
@@ -117,15 +138,19 @@ namespace dynamixel {
 
             // Input
             // get the hardware's state
+            dynamixel_access_mutex.lock();
```

Comment:
I believe that it would be preferable to use trylock (but have little experience in threads) because we would not want the warning messages that are generated when this method takes too much time to execute.